paulmikulskis/redditat
# spamcntrl-landingpage - used firestore api for contact us, added CAle…

…rt.tsx component and added required on input fields.

test:
1. `git fetch origin` and `git checkout john/40-spamcntrl-landing-page-use-firestore-api-and-store-data-on-firestore-db`
2. cd into `spamcntrl-landingpage` and do `yarn install`
3. add the .env content - (EMAIL_USERNAME and EMAIL_PASSWORD)
4. run the spamcntrl-landingpage: `yarn dev`
5. go to localhost:3000/contact and fill out the contact us form
6. click send
7. check out the saved data on firestore database
8. checkout your email (used in the form - should send you an email)

demo:
1. https://youtu.be/40Zagn8BtvM
